src/piprules/namespace_pkgs.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2020 128 Technology, Inc.
+# Copyright 2016 Dillon Giacoppo github.com/dillon-giacoppo
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The contents of this file were mostly copied from:
+# https://github.com/dillon-giacoppo/rules_python_external
+# specifically the files namespace_pkgs.py and wheel.py.
+# 128 Technology has made changes to these files and combined them into this file.
+
+"""Utility functions to discover python package types"""
+import glob
+import os
+import sys
+import textwrap
+
+
+def setup_namespace_pkg_compatibility(wheel_dir):
```

Comment:
did you remove the typehints because this might be run in a python2 environment?

Reply:
Yea, that was my thought, but I can put them back if we want to not support python2 anymore

Reply:
ahh you're right, for some reason I thought this tool was always run in python3 and simply told the target was python2 but that doesn't make sense!